eZ/Publish/Core/Persistence/Legacy/Content/UrlWildcard/Gateway/DoctrineDatabase.php

```diff
@@ -33,9 +38,22 @@ final class DoctrineDatabase extends Gateway
     /** @var \Doctrine\DBAL\Connection */
     private $connection;
 
-    public function __construct(Connection $connection)
+    /**
+     * Criteria converter.
+     *
+     * @var \Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriteriaConverter
+     */
```

Comment:
If you look at the other declarations in the method, it's not consistent. The proper product-wide CS for class members, for versions prior PHP 7.4 is:
```suggestion
    /** @var \Ibexa\Core\Persistence\Legacy\Content\URLWildcard\Query\CriteriaConverter */
```